dist/angular-numbro.js

```diff
@@ -0,0 +1,58 @@
+window.require.define({
+    'angular-numbro': function(exports, require, module) {
```

Comment:
'module' is defined but never used.<br>'require' is defined but never used.<br>'exports' is defined but never used.